### DISCUSSION TITLE
When / How does the `collection.json` file get updated?

### DISCUSSION THREAD

**Discussion by mesqueeb** [SwiftPackageIndex/SwiftPackageIndex-Server#3089] [Q&A]
I've released two new packages yesterday, and I was hoping they'd show in my collection, given they're under my same github account. Yet, looking at the `collection.json`, it hasn't been updated for over 24h now: https://swiftpackageindex.com/mesqueeb/collection.json

Does anyone know the mechanism to this `collection.json` file?

**Comment by finestructure** [comment]
Sorry, I misread your issue. I'll explain shortly with more time.

**Reply by finestructure** [reply]
When I read this yesterday, I thought you were referring to the [`packages.json`](https://github.com/SwiftPackageIndex/PackageList/blob/main/packages.json) file in our PackageList repository.

The `collection.json` file is automatically generated and should include all the three packages listed on your author page. I'll convert this into an issue and take a look. Thanks for the report!
